### DISCUSSION TITLE
How to replace withRouter?

### DISCUSSION THREAD

**Discussion by itamaroh7** [remix-run/react-router#10418] [v5 to v6 Migration]
How to replace withRouter that gets connect / memo function at the export default code line?

example:
export default withRouter(
  connect(mapStateToProps, {
    someFunction
  })(Component)
);

**Comment by dev0T** [comment]
The docs have an example: [here](https://reactrouter.com/en/main/start/faq#what-happened-to-withrouter-i-need-it)

**Reply by itamaroh7** [reply]
I already checked it
thanks anyway
